packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -350,12 +334,15 @@ exit 0
 %{katello_libdir}/__pycache__/uep.*
 %{katello_libdir}/__pycache__/utils.*
 %{katello_libdir}/__pycache__/__init__.*
+%{katello_libdir}/__pycache__/deb_tracer.*
+%{katello_libdir}/__pycache__/scripts.*
 %else
-%attr(750, root, root) %{_sbindir}/katello-package-upload
-%attr(750, root, root) %{_sbindir}/katello-enabled-repos-upload
```

Comment:
This confuses me -- why are we dropping these two scripts?

Reply:
I'm not seeing these two scripts anymore in the newest version https://github.com/Katello/katello-host-tools/tree/4.2.3/bin

Reply:
They're now entry points: https://github.com/Katello/katello-host-tools/blob/a7996744e5ccf565c164449058fa09f7ef5b2f6e/src/setup.py#L37-L43. See https://github.com/Katello/katello-host-tools/commit/8c5b6c1ece2ef7ff76d781a2d5a41443c9ea0845 as well (though it was squash merged, so much less useful to read).

If you properly install using Python packaging tooling it should generate a bin file with the proper shebang etc.

Reply:
I did the bump using ./bump.sh first, and it didn't generated any new line, I had to manually remove and add the new files.

Reply:
It's a major version bump, so it was to be expected that you can't simply use a trivial bump. You should really use `python setup.py install`, but then the RPM packaging equivalents. You may need to rewrite the whole spec. I don't know how well you can use https://pypi.org/project/katello-host-tools/ now or if the RPM still needs some files that aren't part of the package.

My advice would be to turn it into a native Python package and then see which parts are missing.

Reply:
Running py2rpm, I got this output

```
%files -n python3-%{pypi_name}
%doc README.md
%{_bindir}/katello-enabled-repos-upload
%{_bindir}/katello-package-upload
%{_bindir}/katello-tracer-upload
%{python3_sitelib}/apt_plugins
%{python3_sitelib}/dnf_plugins
%{python3_sitelib}/katello
%{python3_sitelib}/yum-plugins
%{python3_sitelib}/zypper_plugins
```